scripts/restart-server

```diff
@@ -78,7 +78,7 @@ logging.info("Stopping server core")
 subprocess.check_call(["supervisorctl", "stop", *core_server_services])
 
 logging.info("Starting server core")
-subprocess.check_call(["supervisorctl", "start", *core_server_services])
+subprocess.check_call(["supervisorctl", "start", *reversed(core_server_services)])
```

Comment:
I think `core_server_services` is just Django and Thumbor, right?  I'm not sure it's important what order we start those in.  (Note that Tornado is done completely separately, above)

Reply:
Currently, yes.  This is mostly future-proofing, and for cleanliness.